src/org/usfirst/frc/team2335/robot/subsystems/Ultrasound.java

```diff
@@ -0,0 +1,47 @@
+package org.usfirst.frc.team2335.robot.subsystems;
+
+import org.usfirst.frc.team2335.robot.Robot;
+
+import edu.wpi.first.wpilibj.Ultrasonic;
+import edu.wpi.first.wpilibj.command.Subsystem;
+
```

Comment:
I'll briefly chime in on how this is handled at my workplace: we mainly ensure that any complex code (function-level documentation, occasionally code within a function if it's notably more complex than the other code around it) has documentation explaining the usage, and that every library we use has similar documentation (or at least is easily findable online).

Since these are external libraries, I'd aim to start building out "external", codebase-level documentation on the broad strokes behind what our code does. Especially cover and answer questions like "What does a particular class do in general", and "why is it implemented and used like it is"? This could even be under a "documentation" directory in the repository, containing text files.